caput/fileformats.py

```diff
@@ -72,11 +72,6 @@ class HDF5(FileFormat):
 
     module = h5py
 
-    @staticmethod
-    def compression_enabled():
-        """Disable compression and chunking due to bug: https://github.com/chime-experiment/Pipeline/issues/33."""
-        return False
-
```

Comment:
How certain are you that we won't want disable compression in the future?  A less invasive change that accomplishes the same thing would be to simply return `True` here (and then not delete the call to this function in mem5h).

It could also permit disabling compression on a case-by-case basis by changing the value returned from this method.  Would that be useful?

Reply:
Do we have a way to enforce minimum HDF5 library version?

If not, an even slicker option would be to check the library version and return True/False based on the needed compression support.